django_x509/static/django-x509/js/switcher.js

```diff
@@ -1,5 +1,21 @@
 django.jQuery(function ($) {
     'use strict';
+    var select_fields = $('.field-certificate, .field-private_key').find(".readonly");
+    // select private_key/certificate field text on click
+    select_fields.on('click', function () {
```

Comment:
try to simplify as follows:

```javascript
$('.field-certificate .readonly, .field-private_key .readonly').on('click', function () {
```

You can remove the previous line in which you declare `select_fields` since those elements are not called elsewhere.

Keep in mind in the javascript world the convention is to use **camelCase** names for variables.